tests/test_TFNetworkLayer.py

```diff
@@ -3065,6 +3065,71 @@ def mask_func(source, **kwargs):
       last_var_v = var_v
 
 
+def test_reduce_mean_in_time():
+  with make_scope() as session:
+    import numpy as np
```

Comment:
Just use `numpy` directly. No need for the `np` alias. And `numpy` is already imported.